### DISCUSSION TITLE
URDF Model Appears Flipped in Foxglove Studio with ROS 2

### DISCUSSION THREAD

**Discussion by 3bdul1ah** [foxglove/community#1063] [Bugs] [answered]
I'm experiencing an issue with my robot visualization:

1. The robot model displays correctly in RViz2
2. However, when viewing in Foxglove Studio, the same model appears flipped
3. The URDF and robot_description topic are working properly

![image](https://github.com/user-attachments/assets/5b940b55-d3aa-4137-aa4f-a95a1bd1ff7f)

Has anyone encountered this behaviour? What might be causing this difference between RViz2 and Foxglove?

**Comment by achim-k** [comment, marked as answer]
Does changing the value for [Mesh up axis](https://docs.foxglove.dev/docs/visualization/panels/3d#scene)  in the 3D panel settings (`Scene -> Mesh "up" axis`) fix it? (Changing the settings requires a refresh)

**Reply by 3bdul1ah** [reply]
Thank you @achim-k , this **worked**!


After reading up on it, seems like if you're using `.dae` files, the `<up_axis>` tag defines the "up" direction for the model (like Z-up or Y-up). In RViz2, this is ignored, but Foxglove actually uses it, so if it's different, the model can appear flipped or weird.

Thanks again! 😊

![image](https://github.com/user-attachments/assets/8c84b371-9fd0-49f1-bc64-32c7e84b8222)

**Reply by jtbandes** [reply]
That's why we have the "Ignore COLLADA `<up_axis>`" setting as well :)
